qnexus/client/circuits.py

```diff
@@ -72,11 +76,12 @@ def get_all(  # pylint: disable=too-many-positional-arguments
         sort=SortFilter.convert_sort_filters(sort_filters),
         page_number=page_number,
         page_size=page_size,
+        scope=scope,
     ).model_dump(by_alias=True, exclude_unset=True, exclude_none=True)
 
     return NexusIterator(
         resource_type="Circuit",
-        nexus_url="/api/circuits/v1beta",
+        nexus_url="/api/circuits/v1beta?scope=global_admin",
```

Comment:
Is this intentional?  `?scope=global_admin` 
Or just leftover from development experiments.